addons/hw_drivers/tools/helpers.py

```diff
@@ -476,7 +476,7 @@ def odoo_restart(delay=0):
 def path_file(filename):
     platform_os = platform.system()
     if platform_os == 'Linux':
-        return Path.home() / filename
+        return Path("/home/pi") / filename  # Path.home() returns odoo user's home instead of pi's
```

Comment:
There are also more usages of this in `KeyboardUSBDriver_L.py` (I guess they can change to use this helper function)

Reply:
right, thank's, it's updated !